azure-pipelines.perf-test.yml

```diff
@@ -0,0 +1,61 @@
+pr: none
+
+trigger: none
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 18.04'
+
+steps:
+  # No need for checkout since we're using build artifacts.
+  - checkout: none
+
+  - template: azure-pipelines.artifacts.yml
+    parameters:
+      artifact: $(artifact)
```

Comment:
Where do these come from in this context?

Reply:
these come from the perf-test build definition variables (sent thru the az pipeline build queue command)